k_gains:
        - 1400.0
        - 1400.0
        - 1400.0
        - 375.0
        - 375.0
        - 375.0
d_gains:
        - 30.0
        - 30.0
        - 30.0
        - 2.0
        - 2.0
        - 2.0
i_gains:
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0
# k_gains:
#         - 1000.0
#         - 900.0
#         - 900.0
#         - 50.0
#         - 50.0
#         - 50.0
# d_gains:
#         - 10.0
#         - 10.0
#         - 10.0
#         - 1.0
#         - 1.0
#         - 1.0
# i_gains:
#         - 4.0
#         - 4.0
#         - 4.0
#         - 4.0
#         - 4.0
#         - 4.0